docker-compose.yml

```diff
@@ -3,20 +3,18 @@ version: '3.1'
 services:
   redis:
     image: redis
-    ports: ['6379:6379']
     healthcheck:
       test: ["CMD", "redis-cli","ping"]
 
   postgres:
     image: postgres:16
-    ports: ['5432:5432']
```

Comment:
These ports don't need to be exposed to the host machine and can sometimes cause conflicts.